src/ButtonThread.cpp

```diff
@@ -124,6 +126,9 @@ int32_t ButtonThread::runOnce()
         switch (btnEvent) {
         case BUTTON_EVENT_PRESSED: {
             LOG_BUTTON("press!\n");
+#if  defined(M5STACK_COREBASIC) 
+            M5.Speaker.tone(1000, 100);
+#endif
 #ifdef BUTTON_PIN
```

Comment:
Tones with button presses is a new feature. Please make sure either all platforms with tone support can use these or better apply them in a separate Pull Request.

Reply:
All platforms that do not support tone can use these, and I will delete this one in the new submission

Reply:
I will delete this feature in the new submission and do not support other platforms for the time being